src/util/json_expr.cpp

```diff
@@ -259,7 +262,7 @@ json_objectt json(
       if(expr.get(ID_value)==ID_NULL)
         result["data"]=json_stringt("NULL");
     }
-    else if(type.id()==ID_bool)
+    else if(type.id()==ID_bool || type.id()==ID_c_bool)
```

Comment:
Remove this for now. We will need some language-specific output functions to produce the correct constants (true/false/null/etc) for each language.